requirements/constraints.txt

```diff
@@ -119,6 +119,10 @@ openai<=0.28.1
 # https://github.com/openedx/edx-platform/issues/34103
 optimizely-sdk<5.0
 
+# lxml 5.1.0 introduced a breaking change in unit test shards
+# This constraint can probably be removed once lxml==5.1.1 is released on PyPI
```

Comment:
Can you update the comment to not unpin this until we reach Python 3.11, so no one unpins it before that? With the current comment, it is ready for removal.

Reply:
updated.